build/yamls/base/provisioning/datasources/create_table.sh

```diff
@@ -163,19 +163,27 @@ clickhouse client -n -h 127.0.0.1 <<-EOSQL
         sourcePodNamespace,
         destinationPodNamespace;
 
-    CREATE MATERIALIZED VIEW flows_policy_view
+    CREATE MATERIALIZED VIEW IF NOT EXISTS flows_policy_view
     ENGINE = SummingMergeTree
     ORDER BY (
         timeInserted,
         flowEndSeconds,
         flowEndSecondsFromSourceNode,
         flowEndSecondsFromDestinationNode,
-        egressNetworkPolicyName,
```

Comment:
Why we remove the `egressNetworkPolicyName` column and add `egressNetworkPolicyNamespace` column instead?